forge-gui/res/cardsfolder/upcoming/astarions_thirst.txt

```diff
@@ -0,0 +1,8 @@
+Name:Astarion's Thirst
+ManaCost:3 B
+Types:Instant
+A:SP$ ChangeZone | ValidTgts$ Creature | TgtPrompt$ Select target creature | Origin$ Battlefield | Destination$ Exile | RememberChanged$ True | SubAbility$ DBPutCounter | SpellDescription$ Exile target creature. Put X +1/+1 counters on a commander creature you control, where X is the power of the creature exiled this way.
```

Comment:
FYI `TgtPrompt$ Select target creature | ` is not needed - Forge will parse this without help

Fine if you want to leave it though